models/calendar.yml

```diff
@@ -359,7 +359,12 @@
     - 9:00 PM:
         type: Talk
         emoji: 🤔
-        title: TBD
+        title: Web Animations
+        description: |
+        With so many different ways to animate the web, it can be hard 
```

Comment:
If it helps, you can clone the repo to your local machine and run it with `npm start` :)